webapp/src/components/card/subscription/index.tsx

```diff
@@ -69,13 +60,7 @@ const SubscriptionCard = ({handleDeleteSubscrption, subscriptionDetails: {channe
                         showFilter && (
                             <div className='d-flex align-item-center margin-left-5'>
                                 <div className='card-filter'>
-                                    <SVGWrapper
-                                        width={14}
-                                        height={14}
-                                        viewBox='0 0 12 12'
-                                    >
-                                        {pluginConstants.SVGIcons.cardFilter}
-                                    </SVGWrapper>
+                                    <i className='azure-devops-icon azure-devops-icon-filter'/>
```

Comment:
In the case of font icons, we can increase accessibility. Please have a look - https://fontawesome.com/docs/web/dig-deeper/accessibility